src/components/plots/scatterPlot.ts

```diff
@@ -16,7 +16,11 @@ export module Plots {
     constructor() {
       super();
       this.classed("scatter-plot", true);
-      this.animator(Plots.Animator.MAIN, new Animators.Base().duration(250).delay(5));
+      var animator = new Animators.Base();
+      animator.startDelay(5);
+      animator.stepDuration(250);
+      animator.maxTotalDuration(600);
```

Comment:
I can create a new variable on the base animator called `RECOMMENDED_MAX_TOTAL_DURATION`. I would want to avoid creating a new animator for just one variable. It does not really make sense.